apps/opik-backend/src/main/java/com/comet/opik/infrastructure/http/CorsFactory.java

```diff
@@ -21,9 +25,10 @@ public class CorsFactory {
             HttpHeaders.ACCEPT,
             HttpHeaders.X_REQUESTED_WITH,
             HttpHeaders.ORIGIN,
-            "Comet-Sdk-Api",
-            "comet-username",
-            "comet-react-ver",
+            COMET_SDK_API_HEADER,
+            COMET_USERNAME_HEADER,
+            COMET_REACT_VER_HEADER,
+            COMET_WORKSPACE_REQUEST_HEADER,
```

Comment:
Should this list cover only request headers or also response headers?  Because we also return:

```
Opik-User-Limit
Opik-User-Remaining-Limit
Opik-User-Remaining-Limit-TTL-Millis
```

Reply:
Only request headers